x-pack/plugins/observability_solution/observability_ai_assistant/server/service/client/adapters/bedrock_claude_adapter.ts

```diff
@@ -222,9 +227,33 @@ export const createBedrockClaudeAdapter: LlmApiAdapterFactory = ({
           throw createInternalServerError(value.modelStreamErrorException.originalMessage);
         }
       }),
+      // map((value) => {
+      //   // JSON.parse(
+      //   //   Buffer.from(JSON.parse(toUtf8(value.chunk.body)).bytes, 'base64').toString('utf-8')
+      //   // );
+
+      //   console.log('bedrock value', value);
+      //   // if (isTokenCountMetric(value)) {
+      //   //   return {
+      //   //     type: StreamingChatResponseEventType.TokenCount,
+      //   //     tokens: {
+      //   //       completion: value.completionTokenCount,
+      //   //       prompt: value.promptTokenCount,
+      //   //       total: value.completionTokenCount + value.promptTokenCount,
+      //   //     },
+      //   //   };
+      //   // }
+      //   return value;
+      // }),
       filter((value): value is BedrockChunkMember => {
         return 'chunk' in value && value.chunk?.headers?.[':event-type']?.value === 'chunk';
       }),
       processBedrockStream({ logger, functions })
     ),
 });
+
+type TokenCountMetric = any;
+
+function isTokenCountMetric(value: BedrockStreamMember): value is TokenCountMetric {
```

Comment:
is this used somewhere?

Reply:
Sorry, bedrock connector is not ready yet